pymatgen/core/structure.py

```diff
@@ -2713,6 +2713,10 @@ def to(self, filename: str = "", fmt: str = "", **kwargs) -> str:
             writer.write_file(filename)
         return str(writer)
 
+    def to_file(self, filename: str = "", fmt: str = "", **kwargs) -> str:
+        """An alias for .to() for more user-friendliness."""
+        return self.to(filename, fmt, **kwargs)
```

Comment:
I think we can move this to `SiteCollection` so only need a single alias and forward both `*args` and `**kwargs`?